data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -405,36 +398,42 @@ def transform(self) -> None:
         )
 
         # Calculate some demographic information.
-        df[self.BLACK_FIELD_NAME] = df["B02001_003E"]
-        df[self.AMERICAN_INDIAN_FIELD_NAME] = df["B02001_004E"]
-        df[self.ASIAN_FIELD_NAME] = df["B02001_005E"]
-        df[self.HAWAIIAN_FIELD_NAME] = df["B02001_006E"]
-        df[self.TWO_OR_MORE_RACES_FIELD_NAME] = df["B02001_008E"]
-        df[self.NON_HISPANIC_WHITE_FIELD_NAME] = df["B03002_003E"]
-        df[self.HISPANIC_FIELD_NAME] = df["B03003_003E"]
+        df[field_names.BLACK_FIELD_NAME] = df["B02001_003E"]
```

Comment:
Can we change this to a rename?

Reply:
good point! will do. leave it better than you found it.